ddl.lua

```diff
@@ -146,8 +146,13 @@ local function get_schema()
     }
 end
 
+local ddl_manager = require('cartridge.ddl-manager')
 return {
     check_schema = check_schema,
     set_schema = set_schema,
     get_schema = get_schema,
+    _section_name = ddl_manager._section_name,
+    _example_schema = ddl_manager._example_schema,
+    apply_schema_from_yaml = ddl_manager.apply_schema_from_yaml,
```

Comment:
I thought again on the matter of the new "yaml" methods -- it seems not very comfortable for users. I think that supporting YAML in existing `set_schema` and `check_schema` methods is better (a YAML is a string, and now they support only a table as input).